packages/@orchidui-vue/src/Overlay/ConfirmationModal/OcConfirmationModal.vue

```diff
@@ -100,7 +105,9 @@ const emitModelValue = (e) => {
       :icon-class="iconClass"
       :variant="variant"
       :description="description"
+      :description-style="descriptionStyle"
       :icon="icon"
+      :show-icon="hideIcon"
```

Comment:
Should here be hideIcon or showIcon cause in props in Confirmation component are init hideIcon but here you pass showIcon

Reply:
Ohh, sorry should be hide icon. I will fix